src/App.ts

```diff
@@ -695,6 +695,8 @@ export default class App {
     } catch (error) {
       this.logger.warn('Authorization of incoming event did not succeed. No listeners will be called.');
       error.code = 'slack_bolt_authorization_error';
+      // disabling due to https://github.com/typescript-eslint/typescript-eslint/issues/1277
+      // eslint-disable-next-line consistent-return
```

Comment:
TL;DR https://github.com/typescript-eslint/typescript-eslint/issues/1277 core eslint consistent-return is not type aware, and typescript-eslint has not implemented an extension to avoid this issue (functions that return `Promise<void>` are not honoured in terms of return type)

Reply:
The clear comment about the reason here is really nice 👍